USER_MANUAL.md

```diff
@@ -452,7 +452,7 @@ The late connecing client has validated the local state before sending the "Pass
   * Players are cleaned up correctly when they disconnect.
 * NOTE: This gym can only be run manually (This can be automated once the test frameworks supports client disconnects [UNR-529](https://improbableio.atlassian.net/browse/UNR-529)).
 * Pre-test steps:
-  * In the Unreal Editor's Content Browser, locate `Content/Maps/PlayerDisconnectGym` and double click to open it.
+  * In the Unreal Editor's Content Browser, locate `Content/Maps/SpatialPlayerDisconnectGym` and double click to open it.
```

Comment:
I can't find this map in my local instance. I've run test map generation. Am I missing something?